### PR TITLE
update doc for bigscape

### PR DESCRIPTION
**Pull request by CunliangGeng**
To update the docs for the latest `run_bigscape`. 

You could see the effect on
- https://nplinker.github.io/nplinker/latest/concepts/config_file/
- https://nplinker.github.io/nplinker/latest/concepts/bigscape/
- https://nplinker.github.io/nplinker/latest/api/bigscape/#nplinker.genomics.bigscape.run_bigscape

### CODE REVIEW

**Comment by CunliangGeng** [comment]
* **#270** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/270?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @CunliangGeng and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/concepts/bigscape.md [hunk 1]
```diff
@@ -1,19 +1,6 @@
 NPLinker can run BigScape automatically if the `bigscape` directory does not exist in the working directory.
+Both BigScape version 1 and 2 are supported.
```
Comment (gcroci2) [inline docs/concepts/bigscape.md]:
```suggestion
Both version 1 and version 2 of BigScape are supported.
```

File: src/nplinker/data/nplinker.toml [hunk 2]
```diff
@@ -46,19 +46,23 @@ version = "3.1"
...
-# automatically configure them.
-# If parameter `--mibig` is set, make sure setting the config `mibig.to_use` to true and
-# `mibig.version` to the version of mibig in bigscape.
+# Required bigscape parameters.
+# --------------
+# For version 1:
+# -------------
+# Required parameters are: `--mix`, `--include_singletons` and `--cutoffs`. NPLinker needs them to run the analysis properly.
+# Do NOT set these parameters: `--inputdir`, `--outputdir`, `--pfam_dir`. NPLinker will automatically configure them.
+# If parameter `--mibig` is set, make sure setting the config `mibig.to_use` to true and `mibig.version` to the version of mibig in bigscape.
```
Comment (gcroci2) [inline src/nplinker/data/nplinker.toml]:
```suggestion
# If parameter `--mibig` is set, make sure to set the config `mibig.to_use` to true and `mibig.version` to the version of mibig in BiG-SCAPE.
```

File: src/nplinker/data/nplinker.toml [hunk 3]
```diff
@@ -46,19 +46,23 @@ version = "3.1"
 # Version of BiG-SCAPE to run. Make sure to change the parameters property below as well
 # when changing versions.
 version = 1
-# Required bigscape parameters are `--mix`, `--include_singletons` and `--cutoffs`. NPLinker needs
-# them to run the analysis properly.
-# Parameters that must NOT exist: `--inputdir`, `--outputdir`, `--pfam_dir`. NPLinker will
-# automatically configure them.
-# If parameter `--mibig` is set, make sure setting the config `mibig.to_use` to true and
-# `mibig.version` to the version of mibig in bigscape.
+# Required bigscape parameters.
```
Comment (gcroci2) [inline src/nplinker/data/nplinker.toml]:
```suggestion
# Required BiG-SCAPE parameters.
```

**Review by gcroci2 — APPROVED** [review]

**Comment by CunliangGeng** [comment]
### Merge activity

* **Jul 29, 8:02 AM EDT**: @CunliangGeng started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/NPLinker/nplinker/270).
* **Jul 29, 8:02 AM EDT**: @CunliangGeng merged this pull request with [Graphite](https://app.graphite.dev/github/pr/NPLinker/nplinker/270).

**Pull request merged**